package.json

```diff
@@ -50,6 +54,7 @@
     "create-reminder-signup": "yarn build && node target/create-reminder-signup/lambda/local.js",
     "next-reminders": "yarn build && node target/next-reminders/lambda/local.js",
     "signup-exports": "yarn build && node target/signup-exports/lambda/local.js",
-    "package": "ARTEFACT_PATH=$PWD/target VERBOSE=true riffraff-artefact"
+    "package": "ARTEFACT_PATH=$PWD/target VERBOSE=true riffraff-artefact",
+    "test": "jest --maxWorkers=1"
```

Comment:
If we wanted more than one worker, we'd have to set up a database for each worker to avoid them clashing. I think the tests run more than fast enough that just the one worker isn't really an issue!